spec/services/cfe/regular_transactions_payload_service_spec.rb

```diff
@@ -100,5 +100,19 @@
         expect(payload[:regular_transactions]).to be_nil
       end
     end
+
+    context "when the 'household flow' feature flag is active", :household_section_flag do
+      let(:session_data) do
+        {
+          "housing_payments_value" => nil,
+          "housing_payments_frequency" => nil,
+        }
+      end
+
+      it "does not send the housing payments deductions, as they are in another screen" do
```

Comment:
I think this test is redundant, and it's my fault for sending you banking up the wrong tree. The issue is that actually RegularTransactionsPayloadService doesn't _care_ which screen housing payments deductions were entered in, because the way you've written this code, they always get stored in the same place in the session. So the reason in this test that nothing is added to the payload is nothing to do with the feature flag, and everything to do with the face you've said that the session contains nil values.

Given that this class is unaffected by the new screens, this test is just a bit confusing and redundant. We can remove it.